frontend/src/game/scenes/Collect.ts

```diff
@@ -34,6 +34,7 @@ export class Collect extends Scene {
     newClickyBit.on("pointerdown", () => {
       EventBus.emit("add-bit");
       newClickyBit.removeFromDisplayList();
+      newClickyBit.removeInteractive();
```

Comment:
Does removing the objects from the various lists actually destroy the object? Like if they are removed are they eventually destroyed from the `clickyBits` list? I don't know enough about Phaser to answer that question myself. If it doesn't this might start to cause a memory leak where the `clickyBits` list contains bits that are no longer interactable. Play the game long enough and it might crash it!